crates/iota-open-rpc/spec/openrpc.json

```diff
@@ -11709,7 +11709,9 @@
             ],
             "properties": {
               "TransactionKind": {
-                "type": "string"
+                "type": "integer",
+                "format": "uint8",
+                "minimum": 0.0
```

Comment:
Is it supposed to be a float format for this? Seems odd but I don't know much about openapi specs

Reply:
Other integer value have it too, so I assume it's fine: https://github.com/iotaledger/iota/blob/d993a49b070008e977cfff3059096d726686e1d5/crates/iota-open-rpc/spec/openrpc.json#L5696-L5731

Reply:
It's weird though, right? I mean I won't hold the PR up for this but I wonder if we should just change it.

Reply:
Hmm, as it's generated I guess we would better need to update the iota-open-rpc crate, but not sure what kind of changes this would require.

Reply:
Oh I didn't realize this was generated. In that case let's not worry about it.